### PR TITLE
Show price after trade by adjusting the probability bars

### LINKED ISSUE #310: Show price after trade by adjusting the probability bars
Instead of showing a column with the price after trade, adjust the bars to reflect this new probability.

Sight does this already. It also shows the change in percentage, but let's leave that for another issue.

Sight: https://drive.google.com/open?id=1rQs-ovKNSyfjwskjRH6GpSZXAI8OUpbQ

### PR DESCRIPTION
**Pull request by mariano-aguero**
closes #310 

### Includes
- Remove price after trade column
- Adjust probabilities regarding amount enter by the user
- Update tests

You can see a demo of the feature right here

https://www.loom.com/share/977daed6a7974767ba66064bdd21df76

@gabitoesmiapodo please check if the transition is OK for you

### CODE REVIEW

File: app/src/components/market/market_fund.tsx [hunk 1]
```diff
@@ -163,20 +163,19 @@ const MarketFundWrapper: React.FC<Props> = (props: Props) => {
     ? `You don't have enough collateral in your balance.`
     : ''
 
+  const probabilities = balances.map(balance => Math.round((balance.probability / 100) * 100))
```
[the remark this answers is not in the document]
Comment (mariano-aguero) [reply app/src/components/market/market_fund.tsx]: Removed, unnecesary code

File: app/src/components/market/market_buy.tsx [hunk 2]
```diff
@@ -95,12 +95,17 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
       )
       const pricesAfterTrade = MarketMakerService.getActualPrice(balanceAfterTrade)
 
-      return [tradedShares, pricesAfterTrade]
+      const probabilities = pricesAfterTrade.map(priceAfterTrade => {
+        const probabilityForPrice = priceAfterTrade * 100
+        return Math.round((probabilityForPrice / 100) * 100)
```
[the remark this answers is not in the document]
Comment (mariano-aguero) [reply app/src/components/market/market_buy.tsx]: Removed, unnecesary code

File: app/src/components/common/outcome_table/index.tsx [hunk 3]
```diff
@@ -115,7 +111,7 @@ export const OutcomeTable = (props: Props) => {
             <BarDiagram
               outcomeName={outcomeName}
               isWinning={isWinning}
-              probability={probability}
+              probability={+probability.toFixed(2)}
```
Comment (mariano-aguero) [inline app/src/components/common/outcome_table/index.tsx]: Mmmm without the method `toFixed`, looks like this...
Screenshot_20200106_164654

**Comment by mariano-aguero** [comment]
@fvictorio I update the base branch to master

File: app/src/components/common/outcome_table/index.tsx (hunk 3 above)
Comment (fvictorio) [reply app/src/components/common/outcome_table/index.tsx]: It seems to me that `BarDiagram` should be responsible of trimming the number of digits.

**Pull request merged**